stored_requests/backends/http_fetcher/fetcher.go

```diff
@@ -34,20 +39,32 @@ import (
 //     "imp3": null // If imp3 is not found
 //   }
 // }
+// or
+// {
+//   "accounts": {
+//     "acc1": { ... config data for acc1 ... },
+//     "acc2": { ... config data for acc2 ... },
+//   },
+// }
 //
 //
 func NewFetcher(client *http.Client, endpoint string) *HttpFetcher {
 	// Do some work up-front to figure out if the (configurable) endpoint has a query string or not.
 	// When we build requests, we'll either want to add `?request-ids=...&imp-ids=...` _or_
-	// `&request-ids=...&imp-ids=...`, depending.
+	// `&request-ids=...&imp-ids=...`.
+
 	urlPrefix := endpoint
 	if strings.Contains(endpoint, "?") {
 		urlPrefix = urlPrefix + "&"
 	} else {
 		urlPrefix = urlPrefix + "?"
 	}
 
-	glog.Info("Making http_fetcher which calls GET " + urlPrefix + "request-ids=%REQUEST_ID_LIST%&imp-ids=%IMP_ID_LIST%")
+	if url, err := url.Parse(endpoint); err != nil {
+		glog.Warningf(`Invalid endpoint "%s": %v`, endpoint, err)
```

Comment:
Thanks for adding this. Although, please consider moving this before creating `urlPrefix` on L56 because if the `endpoint` isn't valid, there's not much point in appending `?` or `&` to it.

Reply:
Done. Should it be an error instead of warning ?

I intend to explore replacing Endpoint with URL also, perhaps in another PR.

Reply:
That's a good point. I'd say it should be a fatal error because if the endpoint isn't valid, the fetching wouldn't work. I see the DB fetcher also doing the same in case of invalid config. Ideally these things should be part of a validate method that validates the config for stored requests